qt/scientific_interfaces/EnggDiffraction/IEnggDiffGSASFittingPresenter.h

```diff
@@ -1,10 +1,12 @@
 #ifndef MANTIDQTCUSTOMINTERFACES_ENGGDIFFRACTION_IENGGDIFFGSASFITTINGPRESENTER_H_
 #define MANTIDQTCUSTOMINTERFACES_ENGGDIFFRACTION_IENGGDIFFGSASFITTINGPRESENTER_H_
 
+#include "IEnggDiffGSASFittingObserver.h"
+
 namespace MantidQt {
 namespace CustomInterfaces {
 
-class IEnggDiffGSASFittingPresenter {
+class IEnggDiffGSASFittingPresenter : public IEnggDiffGSASFittingObserver {
 
 public:
   virtual ~IEnggDiffGSASFittingPresenter() = default;
```

Comment:
The destructor could be declared `virtual` in the new parent class and removed here.